inst/shiny-server/server/FCEPlot.R

```diff
@@ -30,7 +29,7 @@ render_FV_PER_FUN <- reactive({
   data <- subset(DATA(), algId %in% input$FCEPlot.Algs)
   Plot.FV.Single_Func(data, RTstart = rt_min, RTstop = rt_max, show.CI = input$FCEPlot.show.CI,
                show.mean = input$FCEPlot.show.mean, show.median = input$FCEPlot.show.median,
-               scale.xlog = isolate(input$FCEPlot.semilogx), scale.ylog = isolate(input$FCEPlot.semilogy))
```

Comment:
I forgot why we (probably just me) want to isolate 'input$FCEPlot.semilogx'...

Reply:
This was done by me previously when trying to make sure this figure only updated its data when needed, so the axis-scale does not need to recreate the entire figure. However, for the x-axis, this is needed, since different points on the x-axis need to be evaluated to show an informative figure